### PR TITLE
gojq 0.12.0 (new formula) 

### PR DESCRIPTION
**Pull request by zuisong**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gojq.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  homepage "https://github.com/itchyny/gojq"
+  url "https://github.com/itchyny/gojq/archive/v0.12.0.tar.gz"
+  sha256 "3f280536f53ab6670b89424b0cd295d096b2d2928a5ed5d53bc3a23333051c05"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-s -w", "-trimpath", "-o", bin/"gojq", "cmd/gojq/main.go"
+    prefix.install_metafiles
```
Comment (dawidd6) [inline Formula/gojq.rb:12]:
```suggestion
```

File: Formula/gojq.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  desc "Pure Go implementation of jq"
+  homepage "https://github.com/itchyny/gojq"
+  url "https://github.com/itchyny/gojq/archive/v0.12.0.tar.gz"
+  sha256 "3f280536f53ab6670b89424b0cd295d096b2d2928a5ed5d53bc3a23333051c05"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-s -w", "-trimpath", "-o", bin/"gojq", "cmd/gojq/main.go"
```
Comment (dawidd6) [inline Formula/gojq.rb:11]:
```suggestion
    system "go", "build", *std_go_args, "./cmd/gojq"
```

**Review by dawidd6 — APPROVED** [review]

**Comment by zuisong** [comment]
How to build it on Apple Silicon ?

**Comment by carlocab** [comment]
Whoops. Forgot, sorry. That should've been done here by adding the "CI-force-arm" flag.

Bottle job dispatched: https://github.com/Homebrew/homebrew-core/actions/runs/485634822

**Pull request closed without merging**